src/mbgl/gl/attribute.cpp

```diff
@@ -61,131 +64,163 @@ template class VariableAttributeBinding<float, 4>;
 template <>
 void ConstantAttributeBinding<uint8_t, 1>::bind(Context&, AttributeLocation location, optional<VariableAttributeBinding<uint8_t, 1>>& oldBinding, std::size_t) const {
     assert(location != 0);
-    oldBinding = {};
-    MBGL_CHECK_ERROR(glDisableVertexAttribArray(location));
+    if (oldBinding) {
+        MBGL_CHECK_ERROR(glDisableVertexAttribArray(location));
+        oldBinding = {};
+    }
```

Comment:
Right, so revert all of these changes, leaving only the `oldBinding = *this;` change?